cmd/oras/root/blob/delete.go

```diff
@@ -37,6 +37,8 @@ type deleteBlobOptions struct {
 	option.Descriptor
 	option.Pretty
 	option.Target
+	// Deprecated: verbose is deprecated and will be removed in the future.
```

Comment:
Didn't @shizhMSFT make a comment in a previous PR about not commenting private members? That potentially would apply here as well.

Reply:
removed this field after refactoring